doc/man3/OPENSSL_version_text.pod

```diff
@@ -0,0 +1,186 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_VERSION_MAJOR, OPENSSL_VERSION_MINOR, OPENSSL_VERSION_PATCH,
+OPENSSL_VERSION_PRE_RELEASE, OPENSSL_VERSION_BUILD_METADATA,
+OPENSSL_version_major, OPENSSL_version_minor,
+OPENSSL_version_patch,
+OPENSSL_version_pre_release, OPENSSL_version_build_metadata
+OPENSSL_version_text, OPENSSL_version_text_full, OPENSSL_info
+- get OpenSSL version number
```

Comment:
We should probably also document how to test that you're not using a deprecated function here. We can't generate warnings for all of them, for instance some are defines and we can't generate a warning when you're using a deprecated define, so you need to define something else that makes those deprecated defines go away.

Reply:
We have that when configuring